lib/tasks/backfill_missing_award_dates.rake

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+namespace :dqt do
+  desc "imports a csv of trainees from a provided csv"
+  task backfill_missing_award_dates: :environment do
+    failed_trainee_ids = []
+    Trainee.awarded.where(awarded_at: nil).where.not(trn: nil).each do |trainee|
+      response = Dqt::RetrieveTeacher.call(trainee: trainee)
+      awarded_at = response.dig("qualified_teacher_status", "qts_date")
+      Trainees::Update.call(trainee: trainee, params: { awarded_at: awarded_at }, update_dtq: false) if awarded_at
```

Comment:
Was the original plan to call the `Dqt::RetrieveAwardJob` here? At the moment, we're not using the job at all.

Reply:
No, we will need to use this job soon to poll for award data from DQT that has been entered via the ITT portal.